### PR TITLE
add calcIndustry_CCS_limits()

### PR DESCRIPTION
**Pull request by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q**
close #479

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -2,8 +2,8 @@ cff-version: 1.2.0
 message: If you use this software, please cite it using the metadata from this file.
 type: software
 title: 'mrremind: MadRat REMIND Input Data Package'
-version: 0.176.1
-date-released: '2024-03-01'
+version: 0.176.1.9001
```
Comment (fbenke-pik) [inline CITATION.cff:5]: Don't forget to update to non-development version number

File: R/calcIndustry_CCS_limits.R [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+#' @author Michaja Pehl
+#'
+#' @importFrom dplyr %>% group_by full_join inner_join left_join mutate pull
+#'     rename select summarise tibble ungroup
+#' @importFrom quitte add_countrycode_ madrat_mule magclass_to_tibble
+#' @importFrom readr read_delim
+#' @importFrom tidyr complete pivot_longer
+#'
+#' @export
+calcIndustry_CCS_limits <- function(
```
Comment (fbenke-pik) [inline R/calcIndustry_CCS_limits.R:61]: where will this be called? will it be used to generate an inputdata file in fullREMIND.R?

**Review by fbenke-pik — APPROVED** [review]

File: R/calcIndustry_CCS_limits.R (hunk 2 above)
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply R/calcIndustry_CCS_limits.R:61]: At a later point yes.  Currently it is still under development.

File: CITATION.cff (hunk 1 above)
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply CITATION.cff:5]: Are those not rejected in the merge?

Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [reply CITATION.cff:5]:
```
if (length(unlist(thisVersion)[-(1:3)])) {
    stop()
}
```

**Pull request merged**